src/controllers/auth.controller.js

```diff
@@ -0,0 +1,43 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import User from '../models/user.js';
+import { RefreshTokenSchema } from '../models/refreshToken.js';
+import handleResponse from './handleResponse.js';
+
+dotenv.config();
+
+const AuthHandler = async (req, res) => {
+	const user = await User.findOneBy({
+		email: req.body.email,
+	});
+	if (!user) {
+		handleResponse(res, 404, { message: res.__('user does not exist') });
+	} else {
+		try {
+			if (await bcrypt.compare(req.body.password, user.password)) {
+				const accessToken = jwt.sign(
+					{ id: user.id },
```

Comment:
Since we will be using role based **Authorization** add `role` in the data to encrypted.